core/src/main/java/inetsoft/web/viewsheet/service/CoreLifecycleService.java

```diff
@@ -1500,34 +1500,15 @@ public void refreshVSAssembly(RuntimeViewsheet rvs, VSAssembly assembly,
                tfmt.getDefaultFormat().setBackgroundValue(Color.WHITE.getRGB() + "");
             }
 
-            if(info instanceof GroupContainerVSAssemblyInfo) {
-               String groupName = info.getAbsoluteName();
-               int prefixIndex = groupName.lastIndexOf('.');
-               String prefix = prefixIndex < 0 ? null : groupName.substring(0, prefixIndex + 1);
-
-               // when a group container's visibility is changed at runtime (e.g. in a script),
-               // the children have not been added in the client. need to call addDeleteVSObject
-               // to make sure they are added to the client app if necessary.
-               for(String childName : ((GroupContainerVSAssemblyInfo) info).getAssemblies()) {
-                  if(prefix != null && !childName.startsWith(prefix)) {
```

Comment:
Are you sure that the prefix stuff is not required? You should try with a group/tab in an embedded viewsheet, also with multiple levels of embedded viewsheets.